src/dist/config.rs

```diff
@@ -4,17 +4,17 @@ use super::manifest::Component;
 use crate::errors::*;
 use crate::utils::toml_utils::*;
 
-pub const SUPPORTED_CONFIG_VERSIONS: [&str; 1] = ["1"];
-pub const DEFAULT_CONFIG_VERSION: &str = "1";
+pub(crate) const SUPPORTED_CONFIG_VERSIONS: [&str; 1] = ["1"];
+pub(crate) const DEFAULT_CONFIG_VERSION: &str = "1";
 
 #[derive(Clone, Debug)]
-pub struct Config {
-    pub config_version: String,
-    pub components: Vec<Component>,
+pub(crate) struct Config {
+    pub(crate) config_version: String,
+    pub(crate) components: Vec<Component>,
```

Comment:
Did we not decide that `pub(crate)` on fields means basically nothing in a `pub(crate)` struct?